assets/js/index.js

```diff
@@ -98,15 +111,18 @@ function formatDate(date) {
 
 function showTasks(tasks) {
   todoList.innerHTML = sortTasksByStatus(sortTasksByTime(tasks))
-    .map(({ text, createdAt, id: taskId, isDone }) =>
+    .map(({ text, createdAt, id: taskId, isDone, updatedAt }) =>
     `<li id="${taskId}" class="todo-item">
-    <input class="checkbox" type="checkbox" ${isChecked(isDone)}/>
-    <p class="todo"  title="${formatDate(createdAt)}">${text}</p>
-    <i class="fa-solid fa-pen-to-square edite-icon"></i>
-    <i class="fa-solid fa-xmark remove-icon"></i>
+    <input id="checkbox-${taskId}" class="task-item-checkbox" type="checkbox" ${isChecked(isDone)}/>
+    <div id="todo-container-${taskId}">
+    <p class="todo" id="todo-${taskId}" title="${formatDate(createdAt)} ${formatDate(updatedAt)}">${text}</p>
+    </div>
+    <span id="edit-icon-${taskId}" class="fa-solid fa-pen-to-square edit-icon">22</span>
+    <span id="remove-icon-${taskId}"class="fa-solid fa-xmark remove-icon">22</span>
```

Comment:
It's not a good idea to have the id for al of the elements. It would be better to have it on the parent level (`li`) and select that id instead. But This is ok. You don't have to change anything here.

Reply:
Ok.